test/helpers.py

```diff
@@ -26,7 +26,8 @@ def deploy_app_externally(
             **{"PYTHONUTF8": "1"},
         }  # windows apparently needs a bunch of env vars to start python...
 
-    env = {**windows_support, "MODAL_SERVER_URL": servicer.client_addr, **env}
+    env = {**windows_support, "MODAL_SERVER_URL": servicer.client_addr, "MODAL_ENVIRONMENT": "main", **env}
```

Comment:
we hardcode `main` as the environment in `conftest.py`, so this just makes it consistent